tests/test_download.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import asyncio
+from pathlib import Path
+
+import pytest
+
+import coincident
+
+network = pytest.mark.network
+try:
+    import maxar_platform.discovery  # noqa: F401
+
+    not_authenticated = False
+except:  # noqa: E722
+    not_authenticated = True
+maxar_authenticated = pytest.mark.skipif(
+    not_authenticated, reason="Not authenticated with Maxar API"
+)
+
+
+@network
+@maxar_authenticated
+@pytest.mark.filterwarnings("ignore:the actual content type does not match")
+@pytest.mark.skip(reason="temporarily skip to see if CI failing due to async code")
```

Comment:
On a hunch, I disabled this b/c other tests started failing with obscure errors on CI. Running locally I do not see these types of errors, will have to dig into it later, maybe look at how stac-asset does testing (https://github.com/stac-utils/stac-asset/tree/main/tests)

```
>                   warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))
E                   pytest.PytestUnraisableExceptionWarning: Exception ignored in: <function _SelectorTransport.__del__ at 0x10251a200>
E                   
E                   Traceback (most recent call last):
E                     File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/selector_events.py", line 710, in __del__
E                       _warn(f"unclosed transport {self!r}", ResourceWarning, source=self)
E                   ResourceWarning: unclosed transport <_SelectorSocketTransport fd=28>
```